pkg/operation/etcdencryption/util.go

```diff
@@ -0,0 +1,23 @@
+package encryptionconfiguration
+
+import "sort"
+
+func slicesContainSameElements(a, b []string) bool {
+	if len(a) != len(b) {
```

Comment:
```suggestion
	return reflect.DeepEqual(a, b)
```

Reply:
What I wanted to achieve here is to check whether two slices contain the same elements but regardless of the order of the individual elements. I think that DeepEqual takes the ordering into account ...

Reply:
Hm, true.. @adracus is there an easier way to test that?

Reply:
`sets.NewString(a...).Equal(sets.NewString(b...))`

Reply:
great, this is much shorter than my clumsy code ... changed it accordingly and thanks for the hint.